packages/gestalt/src/FormLabel.css

```diff
@@ -0,0 +1,4 @@
+.formLabel {
+  composes: paddingX2 from "./boxWhitespace.css";
+  padding-bottom: 8px;
```

Comment:
Not using `Box` in this case since we can't set asymmetrical padding. We want only `padding-bottom` for the following reasons:

* Components in gestalt should not have extra spacing around them (reason why we don't make it symmetrical)
* Margin bottom is not good enough - it results in "flickering" when you hover from the label to the form field

Reply:
Example of the "flickering" behavior
![flickering-margin-bottom](https://user-images.githubusercontent.com/127199/75571434-0f3e3b00-5a0e-11ea-9d13-b37aa0c8c61d.gif)

Reply:
That flickering is subtle! Good catch